content/03-reference/01-tools-and-interfaces/02-prisma-client/02-constructor.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: 'Constructor'
+metaTitle: 'Prisma Client Constructor (Reference)'
+metaDescription: 'This page describes the Prisma Client constructor parameters.'
+---
+
+## Overview
+
+This page describes the Prisma Client constructor parameters.
+
+## `datasources`
+
+Use the `datasources` constructor parameter to programatically override the URL of a datasource - for example, as part of an integration test. To override a datasource:
+
+1. Define a datasource in the `schema.prisma` file. In the following example, there is a single datasource named `my_database`:
+
+   ```prisma
+   datasource my_database {
+     provider = "sqlite"
+     url      = env("DATABASE_URL")
+   }
+   ```
+   
+   The `DATABASE_URL` environment variable defines the following value:
+
+   ```
+   DATABASE_URL="file:./dev.db"
+   ```
+
+   > **Note**: You must re-generate the Prisma Client each time you add or rename a datasource. Datasource names are included in the generated client.
+
+2. Instantiate a client and override the connection string of the `my_database` datasource:
+
+   ```ts
+   import { PrismaClient } from '@prisma/client'
+
+   const prisma = new PrismaClient({
+     datasources: {
+   |   my_database: 'file:./dev_qa.db',
```

Comment:
I think it has some typo error
```suggestion
       my_database: 'file:./dev_qa.db',
```

Reply:
That is to highlight this line I think.

Reply:
Yup that's correct, you are right about it, I locally run it, My bad 😐😃